lib/components/navbar/index.tsx

```diff
@@ -29,7 +29,7 @@ const NavBar: React.FC<NavBarProps> = ({ navLinks, style }) => {
           src="/logo-tag.png"
           priority
           layout="fixed"
-          width={breakpoint && hamburgerMenuSizes.includes(breakpoint) ? '434px' : '506px'}
+          width={breakpoint && hamburgerMenuSizes.includes(breakpoint) ? '300px' : '506px'}
           height={breakpoint && hamburgerMenuSizes.includes(breakpoint) ? '60px' : '70px'}
```

Comment:
The mobile view looks wonky because we have to maintain the aspect ratio for the image. The ratio is 7.23 px width to 1 px in height. So if the height is 40px, the width should be 289px. This is why you have to change the height as well as the width to resize the image